sencha-workspace/SlateTasksStudent/app/view/AppHeader.js

```diff
@@ -19,6 +19,33 @@ Ext.define('SlateTasksStudent.view.AppHeader', {
             align: 'center'
         },
         items: [{
+            xtype: 'combo',
+            itemId: 'student-selector',
+            name: 'StudentSelector',
+            fieldLabel: 'Student',
+            displayField: 'Username',
+            valueField: 'Username',
+            allowBlank: true,
+            queryParam: 'q',
+            margin: '0 10 0 0',
```

Comment:
@themightychris this needed some space on the right side of it. I did some searching but I didn't see a standard class applied to this type of element to add spacing so I just added 10 pixels of margin to the right manually.